lxd/instance.go

```diff
@@ -540,6 +542,8 @@ func autoCreateContainerSnapshots(ctx context.Context, d *Daemon, instances []in
 	return nil
 }
 
+var cSnapshotsPruneRunning = locking.NewPruneLock()
+
 func pruneExpiredContainerSnapshotsTask(d *Daemon) (task.Func, task.Schedule) {
```

Comment:
Lets rename this function to `pruneExpiredInstanceSnapshotsTask`.

Reply:
Fixed.